config/application.rb

```diff
@@ -118,5 +115,21 @@ class Application < Rails::Application
     config.site_version = "2.10.0"
 
     config.lti_settings = Rails.application.config_for(:lti_settings)
+
+    config.action_controller.default_url_options = { }
+    config.action_mailer.default_url_options = {}
+
+    # Set application host for mailer
+    config.action_mailer.default_url_options = { host: ENV['MAILER_HOST'] }
```

Comment:
You should have code here handling the situation when it's not defined as well. People who are upgrading from previous versions won't have it defined in their env

Reply:
Fixed.